jax/random.py

```diff
@@ -57,11 +58,42 @@
     >>> random.uniform(subkey)
     Array(0.10536897, dtype=float32)
 
+.. note::
+
+   Typed key arrays, with element types such as `key<fry>` above, were
+   introduced in 2023. Before then, JAX conventionally represented
```

Comment:
Might be helpful to mention a specific JAX version; e.g. "introduced in 2023 and ready for use by JAX v0.4.16"